### PR TITLE
Add -skipVerify flag to kapacitor cmd to skip SSL verification

### PR DESCRIPTION
**Pull request by rossmcdonald**
###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: cmd/kapacitor/main.go [hunk 1]
```diff
@@ -28,9 +29,11 @@ var (
 )
 
 var defaultURL = "http://localhost:9092"
+var defaultSkipVerify = false
 
 var mainFlags = flag.NewFlagSet("main", flag.ExitOnError)
 var kapacitordURL = mainFlags.String("url", "", "The URL http(s)://host:port of the kapacitord server. Defaults to the KAPACITOR_URL environment variable or "+defaultURL+" if not set.")
+var skipVerify = mainFlags.Bool("skipVerify", false, "Do not use SSL verification.")
```
Comment (nathanielc) [inline cmd/kapacitor/main.go]: Can you document the env var option here?

Comment (rossmcdonald) [inline cmd/kapacitor/main.go]: Changed to:

```
$ kapacitor --help
Usage of main:
  -skipVerify
    	Disable SSL verification (note, this is insecure). Defaults to the KAPACITOR_UNSAFE_SSL environment variable or false if not set.
  -url string
    	The URL http(s)://host:port of the kapacitord server. Defaults to the KAPACITOR_URL environment variable or http://localhost:9092 if not set.
```

**Comment by nathanielc** [comment]
@rossmcdonald This looks great. Thanks. I believe #532 is different and is referring to the alert post action not the Kapacitor CLI tool.

**Comment by rossmcdonald** [comment]
@nathanielc Oops, you're right. I just saw the error and thought it was referring to this. I've removed it from the summary, though.

**Comment by nathanielc** [comment]
@rossmcdonald LGTM If you wanted to fix #532 too it should be simple ;)

**Pull request merged**